Spokestack/SpokestackSpeechRecognizer.swift

```diff
@@ -32,51 +36,44 @@ import CryptoKit
     @objc public init(_ configuration: SpeechConfiguration, context: SpeechContext) {
         self.configuration = configuration
         self.context = context
-        if let apiSecretEncoded = self.configuration.apiSecret.data(using: .utf8) {
-            self.apiKey = SymmetricKey(data: apiSecretEncoded)
-            self.task = URLSession.shared.webSocketTask(with: URL(string: "wss://api.spokestack.io/v1/asr/websocket")!)
-            
-            // construct auth message
-            let bodyDoubleEncoded = """
-            "{\\"format\\": \\"PCM16LE\\", \\"rate\\": \(configuration.sampleRate.description), \\"language\\": \\"en\\", \\"limit\\": 10}"
-            """
-            let body = "{\"format\": \"PCM16LE\", \"rate\": \(configuration.sampleRate.description), \"language\": \"en\", \"limit\": 10}"
-            let bodySigned = HMAC<SHA256>.authenticationCode(for: body.data(using: .utf8)!, using: self.apiKey!)
-            let bodySignature = Data(bodySigned).base64EncodedString()
-            self.initalizeStreamMessage = """
+        let apiSecretEncoded = self.configuration.apiSecret.data(using: .utf8)! // since the string is by definition utf8-representable, this is a safe unwrap
+        self.apiKey = SymmetricKey(data: apiSecretEncoded)
+        self.task = URLSession.shared.webSocketTask(with: URL(string: self.apiURL)!)
```

Comment:
Any reason this URL construction couldn't just go into the class field so it's even clearer that the `!` is safe?